pkgs/servers/misc/navidrome/default.nix

```diff
@@ -41,7 +41,13 @@ buildGoModule {
 
   buildInputs = [ taglib zlib ];
 
-  CGO_CFLAGS = [ "-Wno-return-local-addr" ];
+  ldflags = [
+    "-s" "-w"
```

Comment:
Why is `-s -w` desired here? My understanding is that this strips some stuff out of the binary. It looks like some things do this in nixpkgs, but most go builds do not.

Reply:
A smaller binary is a better binary? I'll just remove them.